### PR TITLE
Modify rule S3616: Adapt to LaYC

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S3616/javascript/rule.adoc [hunk 1]
```diff
@@ -1,31 +1,28 @@
 == Why is this an issue?
 
-The comma operator (``++,++``) evaluates its operands, from left to right, and returns the second one. That's useful in some situations, but just wrong in a ``++switch++`` ``++case++``. You may think you're compactly handling multiple values in the case, but only the last one in the comma-list will ever be handled. The rest will fall through to the default.
+The use of the comma operator and logical OR (||) operator within switch cases is not recommended. The switch statement is designed to evaluate a single expression and compare it against multiple values. When you use the comma operator or logical OR operator within a case, you're essentially trying to match multiple values or conditions simultaneously.
```
Comment (victor-diez-sonarsource) [inline rules/S3616/javascript/rule.adoc:3]: why did you remove the `but only the last one in the comma-list will ever be handled. The rest will fall through to the default.`

Don't you think is good to mention that?

Comment (yassin-kammoun-sonarsource) [reply rules/S3616/javascript/rule.adoc:3]: Yes, it's indeed good to mention that. I will put it back.

Comment (victor-diez-sonarsource) [reply rules/S3616/javascript/rule.adoc:3]: ok, rest seems good

**Review by victor-diez-sonarsource — APPROVED** [review]
